package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dafka-consumer",
-    "version": "7.6.0",
+    "version": "7.6.1",
```

Comment:
No need to bump

Reply:
Are you sure? I think it is not publishing a version if this is not changed